C: Compilation failure due to invalid C90 syntax

The current iteration of `anyascii.c` is not C90 compliant and thus fails to compile:

```
gcc -Wall -Werror -Wunused -Wextra -Wmaybe-uninitialized -Wstrict-prototypes -Wmissing-prototypes -Wdeclaration-after-statement -Wmissing-declarations -Wmissing-format-attribute -Wnull-dereference -Wformat=2 -Wshadow -Wsizeof-pointer-memaccess -std=gnu99 -pthread -O3 -g -Wstack-protector -fno-omit-frame-pointer -fwrapv -fPIC -D_FORTIFY_SOURCE=2 -c -funsigned-char -Wno-format-y2k -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600  -I/usr/local/include anyascii.c
anyascii.c: In function ‘anyascii’:
anyascii.c:1320:9: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
 1320 |         size_t l = (unsigned char) b[lo + 2];
      |         ^~~~~~
cc1: all warnings being treated as errors
make: *** [Makefile:25: anyascii.o] Error 1
```

Trivially fixable by rearranging the declarations.

--- a/impl/c/anyascii.c
+++ b/impl/c/anyascii.c
@@ -1316,9 +1316,10 @@ size_t anyascii(uint_least32_t utf32, const char **ascii) {
 	const char *b = block(blocknum);
 	size_t blen = ((size_t) (unsigned char) b[0]) * 3 + 1;
 	size_t lo = (utf32 & 0xff) * 3 + 1;
+	size_t l, len;
 	if (lo > blen) return 0;
-	size_t l = (unsigned char) b[lo + 2];
-	size_t len = l & 0x80 ? l & 0x7f : 3;
+	l = (unsigned char) b[lo + 2];
+	len = l & 0x80 ? l & 0x7f : 3;
 	if (len <= 3) {
 		*ascii = b + lo;
 	} else {

--- a/src/main/resources/com/anyascii/build/gen/body.c
+++ b/src/main/resources/com/anyascii/build/gen/body.c
@@ -3,9 +3,10 @@ size_t anyascii(uint_least32_t utf32, const char **ascii) {
 	const char *b = block(blocknum);
 	size_t blen = ((size_t) (unsigned char) b[0]) * 3 + 1;
 	size_t lo = (utf32 & 0xff) * 3 + 1;
+	size_t l, len;
 	if (lo > blen) return 0;
-	size_t l = (unsigned char) b[lo + 2];
-	size_t len = l & 0x80 ? l & 0x7f : 3;
+	l = (unsigned char) b[lo + 2];
+	len = l & 0x80 ? l & 0x7f : 3;
 	if (len <= 3) {
 		*ascii = b + lo;
 	} else {

